src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj

```diff
@@ -25,8 +25,8 @@
     <InjectStringSyntaxAttributeOnLegacy>true</InjectStringSyntaxAttributeOnLegacy>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="System.Text.Json" />
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
+    <PackageReference Include="System.Text.Json" VersionOverride="$(SystemTextJson8Version)" />
```

Comment:
So this is basically saying:
- if targeting net462 or netstandard2.0, add a reference to the STJ 8.0.x package
- if targeting net8.0 or net9.0, don't add a package reference, in which case it'll just pick up the STJ from the corresponding platform

Yes? Makes sense.

Reply:
Yep!